app/controllers/documents_controller.rb

```diff
@@ -16,15 +16,44 @@ def show
   def update
     document = Document.find(params[:id])
     document.update_attributes(document_update_params(document))
+    save_base_path!(document)
```

Comment:
Would it be possible to do this in the `update_attributes`? That saves a database call.

Reply:
Good shout - done here https://github.com/alphagov/content-publisher/pull/70/commits/ef414afe615cdd0f23a5c70071f135cbc852431f